app/controllers/idv/hybrid_mobile/capture_doc_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module HybridMobile
+    class CaptureDocController < ApplicationController
```

Comment:
This will be `DocumentCaptureController`, not `CaptureDocController`. We're going to get rid of the `CaptureDoc` naming entirely.